### DISCUSSION TITLE
FAQ/chinese_path

### DISCUSSION THREAD

**Discussion by giscus** [Air-duino/document#27] [comment]
# FAQ/chinese_path

问题描述 在编译项目时，链接(ld)阶段报错，并且报错信息的路径包含乱码，实际应为中文路径名，如下图所示： 报错信息 问题原因 GCC编译器不支持使用中文路径，需要将此包放置到英文路径下 解决方案 开始之前，你需要确保 Arduino IED 处于关闭状态。 新建一个不带中文路径的文件夹 首先，新建一个你需要放置文件的文件夹路径，并且保证路径纯英文，不...

https://arduino.luatos.com/FAQ/chinese_path.html

**Comment by lishuzhou** [comment]
Air001 看芯片手册是有片上 rtc 资源的，Arduino 如何开发呢，有没有相关 demo？
